aiidalab_eln/cheminfo/__init__.py

```diff
@@ -162,9 +159,15 @@ def import_data(self):
         # Choose the data type.
         if self.data_type == "xray":
             if self.file_name.split(".")[-1] == "cif":
```

Comment:
not important, but I personally find `Path(p).suffix` a bit clearer

Reply:
ok, will apply

Reply:
done